<launch>
  <!-- Launch file for velodyne16 VLP-16 LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find ground_fals)/config/velodyne_kitti.yaml" />
  <!-- feature_extract_enable fix to 0 for normal estimation-->
    <param name="point_filter_num" type="int" value="4"/>
    <param name="runtime_pos_log_enable" type="bool" value="0" />
    <node pkg="ground_fals" type="ground_fals" name="ground_fals" output="screen" />

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find ground_fals)/rviz_cfg/ground_fals.rviz" />
    </group>

</launch>